.dotnet/src/Custom/Chat/ChatOutputPredictionKind.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace OpenAI.Chat;
+
+[CodeGenModel("ChatOutputPredictionType")]
+public readonly partial struct ChatOutputPredictionKind
```

Comment:
See my other comment about removing this class from the API surface. We could get away with just making this internal and not having to rename the Content value to StaticContent.